### ISSUE TITLE
Identifying Mislabeled Images in Supervised Learning Utilizing Autoencoder. (arXiv:2011.03667v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#655]
Identifying Mislabeled Images in Supervised Learning Utilizing Autoencoder. (arXiv:2011.03667v1 [cs.CV])<br>
https://ift.tt/2GIYrG2<br>
<p>Supervised learning is based on the assumption that the ground truth in the training data is accurate. However, this may not be guaranteed in real-world settings. Inaccurate training data will result in some unexpected predictions. In image classification, incorrect labels may cause the classification model to be inaccurate as well. In this paper, I am going to apply unsupervised techniques on the training data before training the classification network. A convolutional autoencoder is applied to encode and reconstruct images. The encoder will project the image data on to latent space. In the latent space, image features are preserved in a lower dimension. Samples with similar features are likely to have the same label. Noised samples can be classified in the latent space. Image projections on the feature space are compared with its neighbors. The images with different labels with their neighbors are considered as incorrectly labeled data. These incorrectly labeled data are visualized as outliers in the latent space. After incorrect labels are detected, I will remove the samples which consist of incorrect labels. Thus the training data can be directly used in training the supervised learning network.</p>
<br>
<br>
via cs.CV updates on arXiv.org http://arxiv.org/<br>
